### ISSUE TITLE
Major breaking change released as a patch version (1.0.1)

### ISSUE THREAD

**Issue by aaronlademann-wf** [Daegalus/dart-uuid#26]
Hi there @Daegalus .

The 1.0.1 release of this library (consisting of https://github.com/Daegalus/dart-uuid/commit/e156dfe4ba9bc3bb6738a993e6070958c006fe9f) is a breaking change since the new lowercase `pi` constant is only available in the dart 2 sdk unless a polyfill such as this is used:

```dart
import 'package:dart2_constant/convert.dart' as convert;

class Uuid {
  // This isn't used, I just am propogating to use of TAU over PI - http://tauday.com/tau-manifesto
  static final TAU = 2 * convert.pi;
}
```

The only other way your change could be considered "non-breaking" for consumers is if you bumped the lower bound of your sdk dependency to something resembling `^2.0.0.dev`.

**Comment by Daegalus** [comment]
Thanks for this I will try to fix it up in the next few hours. My apologies for the mix-up.

I am working on a 2.0 release of this library that changes some things up and uses pointycastle for crypto stuff. So I might be able to clean this up. Also I might just get rid of the TAU thing.

**Comment by robbecker-wf** [comment]
Might I ask for a follow up patch release 1.0.2 with the dart2_constant fix? This way, consumers that have specified a range (^1.0.0 for example) would get the new dart 1 compatible 1.0.2 fix and skip the breaking one.

**Comment by Daegalus** [comment]
Thanks for this I will try to fix it up in the next few hours. My apologies for the mix-up.

I am working on a 2.0 release of this library that changes some things up and uses pointycastle for crypto stuff. So I might be able to clean this up. Also I might just get rid of the TAU thing.

**Comment by aaronlademann-wf** [comment]
Agreed - the breaking change would need to be addressed in a `1.x` release in order to resolve existing breakages for people using semver notation like `^1.0.0`.

**Comment by Daegalus** [comment]
I meant I will be making a 1.0.2 fix for the current version and then make sure it's fixed in my 2.0 work. Sorry if I wasn't clear.

**Comment by Daegalus** [comment]
Should be fixed with the release of 1.0.2. I just straight up removed that constant as its not used anywhere and tests pass.

**Comment by aaronlademann-wf** [comment]
Thanks @Daegalus !!!
